molecule/default/molecule.yml

```diff
@@ -4,10 +4,12 @@ dependency:
   name: galaxy
 driver:
   name: docker
-lint:
-  name: yamllint
-  options:
-    config-file: molecule/default/yamllint.yml
+lint: |
+  E=0
+  yamllint -c molecule/default/yamllint.yml . || E=$$?
```

Comment:
I think you'll need to change yamllint.yml to exclude `.tox` and `.venv`, and for now `tests` because we currently do not scan the `tests/` directory:
```
+# TODO: add tests to scanning
+ignore: |
+  .venv
+  .tox
+  tests
```

Reply:
Thank you. I finally use
```yaml
ignore:
  .*/
  tests/
```